implementations/validator/validator.py

```diff
@@ -56,15 +62,14 @@ def make_html(self, results):
 
     def format_response(self, data, fmt):
         if fmt == "html":
-            response.content_type="text/html"
+            response.content_type = "text/html"
             return self.make_html(data)
         else:
-            response.content_type="application/json"
+            response.content_type = "application/json"
             return json.dumps(data)
 
     def do_test(self):
         url = request.query.get('url', '')
-        typ = request.query.get('type', 'manifest')
```

Comment:
The rationale was that it doesn't currently get used?
The intent of the param was to enable validation of AnnotationLists and Collections as well as manifests, I just haven't implemented those yet.